src/experiments.js

```diff
@@ -90,6 +91,32 @@ export function isExperimentOn(win, experimentId) {
 }
 
 /**
+ * Check whether an experiment is on while allowing viewers to force
+ * the experiment state via a viewer URL param of the form:
+ * `e-$experimentId=1` (on) or `e-$experimentId=0` (off).
+ * NOTE: This should only be used if it is needed and if turning the
+ * experiment on or off does not have security implications.
+ * @param {!Window} win
+ * @param {string} experimentId
+ * @return {boolean}
+ */
+export function isExperimentOnAllowUrlOverride(win, experimentId) {
```

Comment:
Ok. I see. Let's leave `TODO`. This is a single experiment for now and it's only read once.